amm/interp/src/main/scala/ammonite/interp/script/AmmoniteBuildServer.scala

```diff
@@ -69,7 +69,7 @@ class AmmoniteBuildServer(
           Dependency.of(
             "org.scalameta",
             "semanticdb-scalac_" + scala.util.Properties.versionNumberString,
-            "4.4.6"
```

Comment:
I think this version should come from a the `build.sc`, e.g. by generating a `Versions` object.

Pros:
* easier discoverable dependency
* runtime version in sync with compile time version
* enables potential automatic update via. scala-steward

Reply:
I agree... Feel free to fix that if you'd like.

Reply:
Ok, I was going to create a `Versions` in `amm.interp`, but I also found `ammonite.Constants` in scope. Should I rather reuse this?

Reply:
I will open a new PR for this, combined with some more `build.sc` cleanup.

Reply:
FTR, here is the PR: #1163